addon/mixins/routes/application.js

```diff
@@ -31,6 +31,13 @@ export default Ember.Mixin.create({
         outlet: outlet,
         parentView: parentView
       });
+
+      // Check if another modal exists then give it focus
+      Ember.run.later(function() {
+        var remainingViewElement = Ember.$('[role="dialog"]:last-child').closest('.ember-view');
```

Comment:
Ok. Well it sounds like it has advantages and benefits. I'm happy to go with whatever you decide.